diff --git a/app/Post.php b/app/Post.php
index 6df6dd798..49dfd71a2 100644
--- a/app/Post.php
+++ b/app/Post.php
@@ -358,14 +358,14 @@ class Post extends Model
 
         $this->updated = ($entry->entry->updated)
             ? toSQLDate($entry->entry->updated)
-            : gmdate(SQL_DATE);
+            : gmdate(MOVIM_SQL_DATE);
 
         if ($entry->entry->published) {
             $this->published = toSQLDate($entry->entry->published);
         } elseif ($entry->entry->updated) {
             $this->published = toSQLDate($entry->entry->updated);
         } else {
-            $this->published = gmdate(SQL_DATE);
+            $this->published = gmdate(MOVIM_SQL_DATE);
         }
 
         if ($delay) {
diff --git a/app/helpers/DateHelper.php b/app/helpers/DateHelper.php
index 78583900c..fba166733 100644
--- a/app/helpers/DateHelper.php
+++ b/app/helpers/DateHelper.php
@@ -75,5 +75,5 @@ function prepareTime($time = false)
 
 function toSQLDate($date)
 {
-    return date(SQL_DATE, strtotime((string)$date));
+    return date(MOVIM_SQL_DATE, strtotime((string)$date));
 }
diff --git a/app/widgets/Chat/Chat.php b/app/widgets/Chat/Chat.php
index 40057a69a..90a72bfb4 100644
--- a/app/widgets/Chat/Chat.php
+++ b/app/widgets/Chat/Chat.php
@@ -549,7 +549,7 @@ class Chat extends \Movim\Widget\Base
                              $query->where('jidfrom', $jid)
                                       ->orWhere('jidto', $jid);
                          })
-                         ->where('published', $prepend ? '<' : '>', date(SQL_DATE, strtotime($date)));
+                         ->where('published', $prepend ? '<' : '>', date(MOVIM_SQL_DATE, strtotime($date)));
 
 
         $messages = $muc
diff --git a/app/widgets/Menu/Menu.php b/app/widgets/Menu/Menu.php
index d01bfa8c0..1638569ef 100644
--- a/app/widgets/Menu/Menu.php
+++ b/app/widgets/Menu/Menu.php
@@ -164,7 +164,7 @@ class Menu extends Base
         if ($page == 0) {
             $count = 0;
             $last = $posts->orderBy('published', 'desc')->first();
-            \App\Cache::c('since', ($last) ? $last->published : date(SQL_DATE));
+            \App\Cache::c('since', ($last) ? $last->published : date(MOVIM_SQL_DATE));
         }
 
         $items = \App\Post::skip($page * $this->_paging + $count)->withoutComments();
diff --git a/app/widgets/Notifications/Notifications.php b/app/widgets/Notifications/Notifications.php
index 936a762ca..b455c827a 100644
--- a/app/widgets/Notifications/Notifications.php
+++ b/app/widgets/Notifications/Notifications.php
@@ -52,7 +52,7 @@ class Notifications extends Base
     public function ajaxRequest()
     {
         Drawer::fill($this->prepareNotifications());
-        \App\Cache::c('notifs_since', date(SQL_DATE));
+        \App\Cache::c('notifs_since', date(MOVIM_SQL_DATE));
         $this->ajaxSetCounter();
         (new Notification)->ajaxClear('comments');
     }
@@ -61,7 +61,7 @@ class Notifications extends Base
     {
         $since = \App\Cache::c('notifs_since');
         if (!$since) {
-            $since = date(SQL_DATE, 0);
+            $since = date(MOVIM_SQL_DATE, 0);
         }
 
         $count = \App\Post::whereIn('parent_id', function ($query) {
@@ -162,7 +162,7 @@ class Notifications extends Base
         ->get();
         $since = \App\Cache::c('notifs_since');
         if (!$since) {
-            $since = date(SQL_DATE, 0);
+            $since = date(MOVIM_SQL_DATE, 0);
         }
 
         $view = $this->tpl();
diff --git a/src/Movim/Bootstrap.php b/src/Movim/Bootstrap.php
index 8df720c70..0e4216dbc 100644
--- a/src/Movim/Bootstrap.php
+++ b/src/Movim/Bootstrap.php
@@ -107,7 +107,7 @@ class Bootstrap
         define('VIEWS_PATH', DOCUMENT_ROOT . '/app/views/');
         define('HELPERS_PATH', DOCUMENT_ROOT . '/app/helpers/');
         define('WIDGETS_PATH', DOCUMENT_ROOT . '/app/widgets/');
-        define('SQL_DATE', 'Y-m-d H:i:s');
+        define('MOVIM_SQL_DATE', 'Y-m-d H:i:s');
 
         define('API_SOCKET', CACHE_PATH . 'socketapi.sock');
 
diff --git a/src/Movim/Daemon/Core.php b/src/Movim/Daemon/Core.php
index 92663f902..6d4a13c88 100644
--- a/src/Movim/Daemon/Core.php
+++ b/src/Movim/Daemon/Core.php
@@ -207,7 +207,7 @@ class Core implements MessageComponentInterface
     private function cleanupDBSessions()
     {
         DBSession::where('active', false)
-            ->where('created_at', date(SQL_DATE, time()-60))
+            ->where('created_at', date(MOVIM_SQL_DATE, time()-60))
             ->delete();
     }
 
