From 76435a08547cd86bc7f89e89349f8bb6823472cf Mon Sep 17 00:00:00 2001
From: Valerian Saliou <valerian@valeriansaliou.name>
Date: Wed, 18 Nov 2015 16:41:40 +0100
Subject: [PATCH] fixes #568

Signed-off-by: Valerian Saliou <valerian@valeriansaliou.name>
---
 server/functions.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/functions.php b/server/functions.php
index 7706bec..7939f2f 100644
--- a/server/functions.php
+++ b/server/functions.php
@@ -1077,7 +1077,7 @@ function keepGet($current, $no_get) {
     foreach($sanitized as &$get_var) {
         $get_var = preg_replace_callback(
             '/^(.*=)(.+)$/',
-            function($m) { $data = urldecode($m[2]); return '&' . $m[1] . urlencode($data); },
+            function($m) { $data = htmlentities(urldecode($m[2])); return '&' . htmlentities($m[1]) . urlencode($data); },
             $get_var
         );
     }
